dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -1091,5 +1091,105 @@ public void VerifyHierarchicalNavigationTopModeMenuItemsSourceDoesNotCrash()
+                menuItem4.Content = "Item 4";
+
+                navView.MenuItems.Add(menuItem1);
+                navView.MenuItems.Add(menuItem2);
+                navView.MenuItems.Add(menuItem3);
+                navView.MenuItems.Add(menuItem4);
+
+                // Use a pane configuration where tooltips are shown.
+                navView.PaneDisplayMode = NavigationViewPaneDisplayMode.Left;
+                navView.IsPaneOpen = false;
+
+                Content = navView;
+                navView.UpdateLayout();
+
+                Verify.AreEqual(null, ToolTipService.GetToolTip(menuItem1), "Item 1's tooltip should have been [null].");
+                Verify.AreEqual(null, ToolTipService.GetToolTip(menuItem2), "Item 2's tooltip should have been [null].");
+                Verify.AreEqual("Custom tooltip", ToolTipService.GetToolTip(menuItem3), "Item 3's tooltip should have been \"Custom tooltip\".");
+                Verify.AreEqual("Item 4", ToolTipService.GetToolTip(menuItem4), "Item 4's tooltip should have been  \"Item 4\".");
+            });
+        }
+
+        [TestMethod]
+        public void VerifyNavigationViewItemToolTipPaneDisplayMode()
+        {
+            if (!PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone5))
+            {
+                Log.Warning("On RS4 and earlier the test needs to be modified slightly.");
+                return;
+            }
+
+            RunOnUIThread.Execute(() =>
+            {
+                var navView = new NavigationView();
+
+                // Item with non-empty string content using in-built tooltip
+                var menuItem1 = new NavigationViewItem();
+                menuItem1.Content = "Item 1";
+
+                // Item with custom tooltip
+                var menuItem2 = new NavigationViewItem();
+                menuItem2.Content = "Item 2";
+                ToolTipService.SetToolTip(menuItem2, "Custom tooltip");
+
+                navView.MenuItems.Add(menuItem1);
+                navView.MenuItems.Add(menuItem2);
+
+                Content = navView;
+
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.Left, true, null, "Custom tooltip");
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.Left, false, "Item 1", "Custom tooltip");
+
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.LeftCompact, true, null, "Custom tooltip");
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.LeftCompact, false, "Item 1", "Custom tooltip");
+
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.LeftMinimal, true, null, "Custom tooltip");
+
+                // Show tooltips again
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.Left, false, "Item 1", "Custom tooltip");
+
+                SetPaneConfigAndVerifyToolTips(NavigationViewPaneDisplayMode.Top, false /* unused */, null, "Custom tooltip");
```

Comment:
This parameter is not unused, it just doesn't make a difference. I would only add /* unused */ when declaring a function, and there is no reference to said parameter, not when calling a function.

Reply:
I could use "irrelevant" instead. Does that work for you?

Reply:
I wouldn't point that out at all, but if you want to point it out, I think "irrelevant" is more spot on.

Reply:
I felt it useful to point out as an explanation why we only test the pane display mode Top configuration with IsPaneOpne=False config and not with the IsPaneOpen=True config as well as all the other pane display modes in that test. Normally, folks looking at these API tests should be aware that IsPaneOpen is irrelevant for Top mode, but someone might not. So this simply acts as a documentation of that fact. And this is more an "inline-style" comment then adding a more lengthy on above the SetPaneConfigAndVerifyToolTips() call.

Reply:
Honestly, I think we should test both configurations here too. After all, the internal implementation of the tooltip logic could forget to ignore this value in DisplayMode=Top and then we wouldn't notice this. To be sure that it behaves as expected, I would argue that we should test both IsPaneOpen values for DisplayMode=Top too.

Reply:
Works for me.